redux/package.json

```diff
@@ -1,22 +1,24 @@
 {
-    "main": "js/app.js",
```

Comment:
The indentation change is not mine directly. I ran `npm install`, and NPM reformatted the file as it sees fit. I don't see the point of changing it, as next `npm install` will do the same thing until we match the indentation to what it expects.